eb_hooks.py

```diff
@@ -744,6 +744,26 @@ def pre_configure_hook_LAMMPS_zen4(self, *args, **kwargs):
     else:
         raise EasyBuildError("LAMMPS-specific hook triggered for non-LAMMPS easyconfig?!")
 
+def pre_configure_hook_nvhpc_x86_64(self, *args, **kwargs):
+    """
+    pre-configure hook for nvhpc
+    - search and replace operations in the ec dict
+    """
+    if self.name == "NVHPC":
+        eprefix = get_eessi_envvar("EPREFIX")
+        new_opts = f'''installdir=%(installdir)s/Linux_x86_64/%(version)s
+            EPREFIX={eprefix}
+            EBROOTGENTOO={eprefix}/usr
```

Comment:
The naming here is misleading, while Compute Canada have a Gentoo module, we do not. I think you could get away without setting it at all and just replace the usage below with `$EPREFIX/usr/lib64`